requirements-dev.txt

```diff
@@ -18,7 +18,8 @@ commonmark==0.9.1
 coverage==6.5.0
 craft-archives==1.1.0
 craft-cli==2.0.0
-craft-parts==1.21.0
+#craft-parts==1.21.0
+git+https://github.com/canonical/craft-parts.git@CRAFT-1770-has-overlay#egg=craft_parts
```

Comment:
noting this to myself: update before merging

Reply:
Craft-parts 1.22.0 released.